### DISCUSSION TITLE
Normalizing Channels for PovertyMap

### DISCUSSION THREAD

**Discussion by niki-amini-naieni** [p-lambda/wilds#109] [Q&A] [answered]
Hello,

In the Wilds codebase, I do not see any normalization of the images loaded from the PovertyMap dataset. However, in appendix E.8.4, in the **data processing** section, the corresponding paper reads "We normalize each channel by the pixel-wise mean and standard deviation for each channel." Has this normalization already been done before loading the PovertyMap data with the provided data loaders in the WILDS codebase?

Link to original paper: https://arxiv.org/pdf/2012.07421.pdf

Thanks!

**Comment by kohpangwei** [comment, marked as answer]
Yup, it's been pre-normalized. See here: https://github.com/p-lambda/wilds/blob/main/wilds/datasets/poverty_dataset.py#L69

and the corresponding dataset processing function that passes in `normalize=True`: https://github.com/p-lambda/wilds/blob/main/dataset_preprocessing/poverty/batcher.py#L63

**Reply by niki-amini-naieni** [reply]
Great, thank you!
